org.opent2t.sample.hub.superpopular/com.hue.hub/js/common.js

```diff
@@ -0,0 +1,22 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
```

Comment:
I'm not sure at the moment, this has been all over the place.  My recommendation is to leave it to be consistent and we can remove it later for everyone if thats the way we want to go.

---
In reply to: [81489009](https://github.com/openT2T/translators/pull/96#discussion_r81489009) [](ancestors = 81489009)